### ISSUE TITLE
Losing all data/contact info when logging out of the app

### ISSUE THREAD

**Issue by niwim** [Pravica-Inc/pravica-issues#7]
**Describe the bug**
Tested to log out of the app and log in again with Wise. Don´t know what is expected behaviour but when I do this I lose:
- All of my contacts
- Message history
- People can send massages to me and I get the notification, but when I open the message I only get *Anonyms' and the messages does not load

**To Reproduce**
1. Go to 'Settings'
2. Click on 'Logout then log in again via Wise'
3. Scroll down to 'N/A'
4. See error 'N/A'

**Expected behavior**
Expected contacts to load and maybe old conversations?
Also expected to be able to receive messages from already accepted contacts

**Screenshots**
![675AA31B-29F7-466D-92BA-6AE99C3B6489_1_105_c](https://user-images.githubusercontent.com/10028990/119092416-b55cd300-ba0e-11eb-8c2c-7e2ba4281187.jpeg)
![D0D7214A-BB41-47E6-AF85-CA637A567434_1_105_c](https://user-images.githubusercontent.com/10028990/119092418-b68e0000-ba0e-11eb-9830-52aeb28b73a2.jpeg)
![7BE2814A-8D8E-42E0-ADF6-38A2ED159B73_1_105_c](https://user-images.githubusercontent.com/10028990/119092420-b7269680-ba0e-11eb-8b27-aa57bf36c8dd.jpeg)

**Smartphone (please complete the following information):**
- Device: iPhone Xs
- OS: iOS 14.5.1
- Version 2.3.1

**Additional context**

**Comment by niwim** [comment]
The conversations and contact returned after about a day. Looks like it can be some kind of delay in the syncing or something.

**Comment by Pravica-Inc** [comment]
Yeah it's a syncing issue, we're going to fix it in the next release
